docs/learnpaths/paths.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_label: Introduction
+sidebar_position: 10
+title: Introduction
+description: Learning paths for different personas.
+keywords: [learning paths]
+doc-persona: [kubewarden-all]
+doc-type: [explanation]
+doc-topic: [learning-paths]
+---
+
+<head>
+  <link rel="canonical" href="https://docs.kubewarden.io/learning-paths/paths"/>
+</head>
+
+If you are new to Kubewarden, these sections contain a set of learning paths for different personas.
+
+|Persona|Description|
+|-|-|
+|[user](kubewarden-user.md)|Someone who takes a policy and uses it in a cluster. They run policies and observe results.|
+|[operator](kubewarden-operator.md)|Someone operating Kubewarden in a cluster. Installs and maintains Kubewarden.|
+|[integrator](kubewarden-integrator.md)|Builds with, using, or on top of KW. Perhaps a custom UI or a helper tool for generating policies.|
+|[distributor](kubewarden-distributor.md)|Someone who has written a policy and wants to share it with others. They want the policy to be easily consumed by the “policy consumer” role. A distributor of Kubewarden policies, to places such as ArtifactHub.|
```

Comment:
Here "policy consumer" is inconsistent with the rest of the table; I would prefer to rename "user" to "policy consumer" or "policy user".